localnet/.gitignore

```diff
@@ -0,0 +1,138 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Just added because I added instructions for setting up a python venv if you want to run tests without the container so to prevent to push the .venv and .pyc files. That file came from https://github.com/github/gitignore/blob/master/Python.gitignore
Should I remove it?